coresimd/x86/sse4a.rs

```diff
@@ -33,9 +33,12 @@ extern "C" {
 ///
 /// If `length == 0 && index > 0` or `lenght + index > 64` the result is
 /// undefined.
+///
+/// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_extract_si64)
```

Comment:
So these intrinsics are part of the AMD intrinsics, and are not in the intel IntrinsicsGuide. AMD only provides a pdf with the spec AFAICT so the correct place to point them at would be:

```
/// [AMD64 Architecture Programmer's Manual, Volume 3: General-Purpose and System Instructions](http://support.amd.com/TechDocs/24594.pdf) 
```